### PR TITLE
Fixed Readme File

### PR DESCRIPTION
**Pull request by Sourav6971**
Fixed minor typos in the Readme.md file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,52 +10,52 @@ More specifically:
 
 ## WHAT
 
-The scipts will create the following files:
+The scripts will create the following files:
 
-* multiple SVG graphics showing different statistcs about the usage of jenkins
-* a file `<plugin-name>.stats.json`for every plugin, containing the following data (example: git-plugin - [git.stats.json](http://stats.jenkins.io/plugin-installation-trend/git.stats.json) ):
+- multiple SVG graphics showing different statistcs about the usage of jenkins
```
Comment (MarkEWaite) [inline README.md:15]:
```suggestion
- multiple SVG graphics showing different statistics about the usage of Jenkins
```

File: README.md [hunk 2]
```diff
@@ -10,52 +10,52 @@ More specifically:
...
-  
-* `installations.json`: the number of jenkins installations by the jenkins core version
-* `capabilities.json`: a reverse cumulation of the `installations.json` to assist plugin developers in choosing base versions for there further plugin development
+  - `name`: the name of the plugin (as used in the filename)
+  - `installations`: the number of installations for a given month
+  - `installationsPercentage`: the percentage of installations
+  - `installationsPerVersion`: the number of plugin installations for for each plugin version
+  - `installationsPercentagePerVersion`: the percentage of installations a specifig version of the plugin makes up
+
+- `installations.json`: the number of jenkins installations by the jenkins core version
```
Comment (MarkEWaite) [inline README.md:24]:
```suggestion
- `installations.json`: the number of Jenkins installations by the Jenkins core version
```

File: README.md [hunk 3]
```diff
@@ -10,52 +10,52 @@ More specifically:
...
+- `capabilities.json`: a reverse cumulation of the `installations.json` to assist plugin developers in choosing base versions for there further plugin development
 
 ## HOWTO
 
 ### Data
 
-1. you need to download the raw data (*.json.gz) from jenkins-ci.org
+1. you need to download the raw data (\*.json.gz) from jenkins-ci.org
```
Comment (MarkEWaite) [inline README.md:31]:
```suggestion
1. you need to download the raw data (\*.json.gz) from jenkins.io
```

**Review by MarkEWaite — DISMISSED** [review]
Minor changes suggested.

**Review by lemeurherve — APPROVED** [review]

**Pull request merged**